person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..900)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Public%20method%20can_use_services%3F), Please make the `can_use_services?` method a public method, not private as you did.
[Hint] You can do that by moving it before `private` in line 14 above.

Reply:
@Omar-Muhamad  thank you 
done